apps/monitoring/helm-base-versions-slack/kustomization.yaml

```diff
@@ -3,17 +3,4 @@ kind: Kustomization
 nameSuffix: "-slack"
 resources:
   - ../helm-base-versions/helm-base-versions.yaml
```

Comment:
I dont think you need this line now, this was here and then being patched, you've created a standalone file for the slack side

Reply:
ooh i see that make sense as it was a patched from the other file